src/pybind/mgr/dashboard/frontend/src/app/shared/services/wizard-steps.service.ts

```diff
@@ -17,6 +17,8 @@ export class WizardStepsService {
   sharedData = new DriveGroup();
   osdDevices: InventoryDevice[] = [];
   osdCapacity = 0;
+  clusterCreation: boolean;
+  hiddenServices: string[] = [];
```

Comment:
maybe these vars shld be deleted if not referenced anywhere

Reply:
These are being referenced in the code.